src/game/models/asteroid.rs

```diff
@@ -144,6 +170,81 @@ impl Asteroid {
             on_screen: false,
         }
     }
+
+    pub fn can_split(&self) -> bool {
+        self.shape.len() > 10
+    }
+
+    pub fn split<P: Positioned>(&mut self, other: &P) -> Vec<Asteroid> {
+        self.normalize_rotation();
+        //let base_speed = self.vel();
+        let index_nearest = self.index_nearest_point(other);
+        let num_pieces = 3;
```

Comment:
Yeah, I was going to do that in a later PR. 2-5 would work well. With a low number of vertices and a high number of chunks, the math for splitting them breaks down a bit, but 5 should still be fine.